### ISSUE TITLE
I can't get the notification.

### ISSUE THREAD

**Issue by square1grp** [notifee/react-native-notifee#251]
```
import notifee from '@notifee/react-native';

export const onMessageReceived = async remoteMessage => {
  console.log(JSON.stringify(remoteMessage));

  const notificationId = await notifee.displayNotification({
    title: 'Test title',
    body: 'Test body',
    android: {
      channelId: 'default'
    }
  });

  console.log(notificationId);
};
```

It returns notificationId, but I can't see any notification.

Is there something wrong with my code???

Please help me...
Thank you.

**Comment by helenaford** [comment]
Hi, have you created the channel first?

**Comment by square1grp** [comment]
no.

**Comment by square1grp** [comment]
I just set channelId as 'default'.

**Comment by helenaford** [comment]
There's an example in the docs which should be able to help you. For android, a channel has to be created first.

**Comment by square1grp** [comment]
ok. let me try again after creating a channel.

**Comment by square1grp** [comment]
I integrated FCM with notifee, now there're 2 notifications for 1 event for the background state.
I think 1 is from FCM and another one is from notifee.
